can-bus/json/FSM/FSM_enum.json

```diff
@@ -1,3 +1,73 @@
 {
-    
+	"BrakeActuation":{
+		"BRAKE_IS_ACTUATED_FALSE_CHOICE": 0,
+		"BRAKE_IS_ACTUATED_TRUE_CHOICE": 1
+	},
+	"BrakePressureSensorOCSC":{
+		"PRESSURE_SENSOR_IS_OPEN_OR_SHORT_CIRCUIT_FALSE_CHOICE": 0,
+		"PRESSURE_SENSOR_IS_OPEN_OR_SHORT_CIRCUIT_TRUE_CHOICE": 1
+	},
+	"BrakePedalOCSC":{
+		"PEDAL_IS_OPEN_OR_SHORT_CIRCUIT_FALSE_CHOICE": 0,
+		"PEDAL_IS_OPEN_OR_SHORT_CIRCUIT_TRUE_CHOICE": 1
+	},
+	"PappsOCSC":{
+		"PAPPS_OCSC_IS_ACTIVE_FALSE_CHOICE": 0,
+		"PAPPS_OCSC_IS_ACTIVE_TRUE_CHOICE": 1
+	},
+	"SappsOCSC":{
+		"SAPPS_OCSC_IS_ACTIVE_FALSE_CHOICE": 0,
+		"SAPPS_OCSC_IS_ACTIVE_TRUE_CHOICE": 1
+	},
+	"AppsDisagreement": {
+		"APPS_HAS_DISAGREEMENT_FALSE_CHOICE": 0,
+		"APPS_HAS_DISAGREEMENT_TRUE_CHOICE": 1
+	},
+	"BrakeAccDisagreement":{
+		"BRAKE_ACC_DISAGREEMENT_FALSE_CHOICE": 0,
+		"BRAKE_ACC_DISAGREEMENT_TRUE_CHOICE": 1
+	},
+	"FlowMeterHasUnderFlow":{
+		"FLOW_METER_HAS_UNDERFLOW_FALSE_CHOICE": 0,
+		"FLOW_METER_HAS_UNDERFLOW_TRUE_CHOICE": 1
+	},
```

Comment:
Since these are all true or false signals, no need to make them an enum. I'd recommend just specifying the signal with `bits: 1` and they'll be generated to be booleans.